src/components/Navigation.jsx

```diff
@@ -1,23 +1,61 @@
-import React, { Component } from "react";
-import Logo from "../assets/ImagineThisLogo.png";
-import Navbar from "react-bootstrap/Navbar";
-import NavDropdown from "react-bootstrap/NavDropdown";
-import Nav from "react-bootstrap/Nav";
-import "../css/guidebar.css";
+import React, {
+    Component
+} from 'react'
+import Logo from "../assets/ImagineThisLogo.png"
+import Navbar from "react-bootstrap/Navbar"
+import NavDropdown from "react-bootstrap/NavDropdown"
+import Nav from "react-bootstrap/Nav"
+import "../css/guidebar.css"
+import Form from "react-bootstrap/Form"
+import FormControl from "react-bootstrap/FormControl"
+import InputGroup from "react-bootstrap/InputGroup"
+import Button from "react-bootstrap/Button"
+import axios from "axios";
+
+
+const host = "http://localhost:8080";
+
 
 /*
  * Top navigation containing links to all external pages
  */
 class Navigation extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {value: ''}
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({value: event.target.value});
+  }
+
+  handleSubmit(event) {
+    var url = host + "/api/v1/projects/" + this.state.value + "/feedback"
+    axios.get(url).then(res => {
+        if (res.status === 200) {
```

Comment:
If the code goes into the .then, it means it was successful, else would go in the catch, there's no need to check for the status code.

Reply:
I advice adding .catch instead of the if statement